src/main/webapp/ts/matches.ts

```diff
@@ -6,6 +6,8 @@ type authInfo = {
 async function loadMatches() {
     const logStatus = await getLogStatus();
 
+    console.log("loadMatches IS BEING CALLED");
```

Comment:
huzzah! Glad it finally got called (I hope) :)
You probably meant to clean this up.

Reply:
oops,  thanks for catching this, I deleted it :)